src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -554,30 +552,28 @@ def add_plane_segmentation(
     segmentation_extractor : SegmentationExtractor
         The segmentation extractor to get the results from.
     nwbfile : NWBFile
-        The nwbfile to add the plane segmentation to.
+        The NWBFile to add the plane segmentation to.
```

Comment:
Here I feel that it should be as it was (`nwbfile`) and not `NWBFile`. My feeling is that the former is the instance and the one in capital we are using it for the type. What do you think? I see that you made this change in other places as well.

Reply:
`NWBFile` has a precise definition within the software libraries, as it is defined by PyNWB. On the other hand, users are free to call instances of `NWBFile` whatever they want. `nwbfile` is a convention we use within neuroconv, but it is not required that the user use or even be aware of this convention. I think it's clear from the sentence "The NWBFile to add the plane segmentation to." that we are talking about an instance of the `NWBFile` class.

Reply:
The convention I am using if if I am talking about an instance of an NWBFile class as an in-memory object, I am calling it "a" or "the NWBFile". If I am talking about a file of the type .nwb, then I call it an "NWB File".

Reply:
This makes sense. Thanks for clarifying.

Reply:
Can you add a blurb about that to the style guide? https://github.com/catalystneuro/neuroconv/blob/main/docs/developer_guide/style_guide.rst